modules/kubernetes/eks-core/modules.tf

```diff
@@ -15,16 +15,16 @@ locals {
     "velero",
     "promtail"
   ]
-  dns_zone = {
+  route53_zones = {
```

Comment:
Hmm why don't you want to use dns zones any more?
From my point of view we are working with the construct of dns zones, not a specific product.
I can live with it but just a thought.

Reply:
Because this local variable does not represent dns_zones as in all other uses. Its an iteration of Route53 zone ids. Having the same name would confuse more than it helps. Giving it a different name makes it clear that it contains other data.